lib/orders/bittrex.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var AbstractOrders = require('./abstract'),
+    request = require('request'),
+    util = require('util'),
+    _ = require('underscore');
+
+function BittrexOrders (client) {
+    var self = this;
+
+    AbstractOrders.apply(this, arguments);
+
+    this.name  = 'bittrex';
+    this.key   = this.name + 'Orders';
+    this.url   = 'https://bittrex.com/api/v1.1/public/getorderbook?market=BTC-LSK&type=both&depth=50';
+    this.limit = 100;
+
+    this.reverse = {
+        bids : true,
+        asks : false
+    };
+
+    this.retrieveOrders = function (cb) {
+        console.log('Orders:', 'Retrieving orders from', self.name + '...');
+
+        request.get({
+            url : self.url,
+            json : true
+        }, function (err, resp, body) {
+            if (err || resp.statusCode !== 200) {
+                return cb(err || 'Response was unsuccessful');
+            }
+
+            var message = body.message;
+            if (message) {
+                return cb(message);
+            } else if (_.isObject(body)) {
+                return cb(null, self.acceptOrders(body));
+            } else {
+                return cb('Invalid data received');
+            }
+        });
+    };
+
+    this.acceptOrders = function (data) {
+        var asks = [],
+            bids = [];
+
+        for(var i in data.result.buy)
```

Comment:
JShint hissues start here, can be resolved by changing to the below for line 49 to like 54
```
        /* jshint shadow:true */
        for(var i in data.result.buy) {
            bids.push([data.result.buy[i].Rate, data.result.buy[i].Quantity]);
        }

        for(var i in data.result.sell) {
            asks.push([data.result.sell[i].Rate, data.result.sell[i].Quantity]);
        }
        /* jshint shadow:false */
```